toolkit/smart-contracts/offchain/src/scripts_data.rs

```diff
@@ -251,7 +260,7 @@ mod tests {
 				version_oracle_validator:
 					"addr_test1wqxm9e576k5ew7g7ctuqx77p9u7zytesnjsx54q2etck00gqplk0l".into(),
 				governed_map_validator:
-					"addr_test1wppqfuvptxq3rwv2qwke2dkh8vd0mmc8k4r6a44k86tpckss2zg4v".into(),
+					"addr_test1wpyjqa4deje4c9jsjzt3djv28zszvatzup60vvjvn5cdqeggfmj6v".into(),
```

Comment:
Please update addresses in dev/envs. We'll have to see how far to propagate it. On staging and ci-preview we should update manually I think.

Reply:
OTOH, chain resets are coming. So, maybe forget about deployments, but lets update addresses.